src/Atmos/Model/precipitation.jl

```diff
@@ -102,32 +104,45 @@ function compute_gradient_flux!(
     diffusive.precipitation.∇q_rai = ∇transform.precipitation.q_rai
 end
 
-function flux_first_order!(
-    precip::RainModel,
-    atmos::AtmosModel,
-    flux::Grad,
-    state::Vars,
-    aux::Vars,
-    t::Real,
-    ts,
-    direction,
-)
+"""
+    TerminalVelocity{PV} <: TendencyDef{Flux{FirstOrder}, PV <: Rain}
+
+TODO: Add docs
```

Comment:
There is a leftover TODO here about missing docs

Reply:
Ah, good catch!

Reply:
Is this a good name for the source? And would you like to push a commit for the doc string?

Reply:
Not sure if it's a good name. What you compute here is a sum of air velocity with which you advect the precipitation plus its own terminal velocity. So I would say it's just a precipitation flux. 

The docs on how terminal velocity is computed are already available in the microphysics parameterization documentation. I don't think there is a need to repeat it here?